### ISSUE TITLE
New Fluent API

### ISSUE THREAD

**Issue by doanduyhai** [doanduyhai/Achilles#182]
```java

// With entityManager
entityManager
.find(10L)
.withConsistencyLevel(...)
.execute();

// With persistenceManager
persistenceManager
.find(User.class, 10L)
.withConsistencyLevel(...)
.execute();

// Async
entityManager
.find(10L)
.withConsistencyLevel(...)
.executeAsync();

//Reactive
entityManager
.find(10L)
.withConsistencyLevel(...)
.executeRx();

```

**Comment by bric3** [comment]
instead of `.executeRx()` I propose to use `.observe()`, it's a nice naming pattern as it return an `Observable`, used in many RxJava presentations from Ben Christensen and others.

**Comment by doanduyhai** [comment]
Ok for `observe()`

**Comment by doanduyhai** [comment]
Fixed in Achilles 4.0.0, support for RxJava to be processed in another Issue
